simple/utils/astrometry.py

```diff
@@ -2,183 +2,167 @@
 from typing import Optional, Union
 from sqlalchemy import and_
 import sqlalchemy.exc
+from simple.schema import Parallaxes
 from astropy.units import Quantity
 from astropy.table import Table
 from astrodbkit2.astrodb import Database
 from astrodb_utils import AstroDBError, find_source_in_db, find_publication
 
 
 __all__ = [
-    "ingest_parallaxes",
+    "ingest_parallax",
     "ingest_proper_motions",
 ]
 
 logger = logging.getLogger("SIMPLE")
 
 
-# PARALLAXES
-def ingest_parallaxes(db, sources, plxs, plx_errs, plx_refs, comments=None):
+def ingest_parallax(
+    db,
+    source: str = None,
+    parallax: float = None,
+    plx_error: float = None,
+    reference: str = None,
+    comment: str = None,
+    raise_error: bool = True,
+):
     """
 
     Parameters
     ----------
     db: astrodbkit2.astrodb.Database
         Database object
-    sources: str or list[str]
-        list of source names
-    plxs: float or list[float]
-        list of parallaxes corresponding to the sources
-    plx_errs: float or list[float]
-        list of parallaxes uncertainties
-    plx_refs: str or list[str]
-        list of references for the parallax data
-    comments: Optional[Union[List[str], str]]
-
-    Examples
-    ----------
-    > ingest_parallaxes(db, my_sources, my_plx, my_plx_unc, my_plx_refs)
+    sources: str
+        source name
+    parallax: float
+        parallax corresponding to the source
+    plx_err: float
+        parallax uncertainties
+    reference: str
+        reference for the parallax data
+    comment: str
+        comments
+    raise_error: bool
+        raise error if there is an error during ingest
```

Comment:
Be explicit
True: raises an error
False: does not raise an error, returns flags dictionary.